demo/src/index.js

```diff
@@ -1,148 +1,151 @@
-    });
-  }
-
-  render() {
-    return (
-      <div className="demo-wrapper">
-        <div className="top-part">
-          <h1>thundra-trace-chart Demos</h1>
-
-          <button onClick={() => {
-            this.sampleTraceHighlights(this.state.spanHighlightsArrIndex + 1);
-          }}>
-            toggle span highlights
-          </button>
-
-          <button onClick={() => {
-            this.sampleActiveSpanIds(this.state.activeSpanIndex + 1);
-          }}>
-            next span
-          </button>
-        </div>
-
-        {/* <button onClick={() => {
+    render() {
+        return (
+            <div className="demo-wrapper">
+                <div className="top-part">
+                    <h1>thundra-trace-chart Demos</h1>
+
+                    <button
+                        onClick={() => {
+                            this.sampleTraceHighlights(
+                                this.state.spanHighlightsArrIndex + 1
+                            )
+                        }}
+                    >
+                        toggle span highlights
+                    </button>
+
+                    <button
+                        onClick={() => {
+                            this.sampleActiveSpanIds(
+                                this.state.activeSpanIndex + 1
+                            )
+                        }}
+                    >
+                        next span
+                    </button>
+                </div>
+
+                {/* <button onClick={() => {
 					const spanId = "9400a8a9-9650-4312-9514-d4bbc1114a97";
 					const firstSpanElement = document.getElementById(spanId);
 					firstSpanElement.scrollIntoView({behavior: "smooth", block: "start", inline: "nearest"});
 				}}>
 					scroll to first span
 				</button> */}
 
-
-        <div className="ttc-wrapper">
-          <ThundraTraceChart
-            traceId="4e81414c-2bff-439f-9e5c-9e6699b4e24b" // TODO: remove traceId from props
-            traceSummary={sampleTrace()}
-            spanDetails={this.sampleTraceDetail()}
-
-            // spanHighlights={this.state.spanHighlightsArr}
-            activeSpanIds={this.state.activeSpanIdsArr}
-            // activeSpanIds={[""]}
-
-
-            // autoScrollOnMount={true} // do we need to add a new prop like this for autoScroll?
-
-            // showHeader={false}
-            // showMiniTrace={false}
-            // showTraceChartHeader={false}
-            // showSpanDetail={false}
-            // showSpanDetailTitle={false}
-
-
-            serviceNameColumnTitle="Service Name"
-            spanInfoColumnTitle="Operation Name"
-
-            onSpanClicked={(spanId) => console.log("span clicked; spanId: ", spanId)}
-          />
-        </div>
-      </div>
-    );
-  }
+                <div className="ttc-wrapper">
+                    <ThundraTraceChart
+                        traceId="382d12cc-b846-3837-ba10-7b9f79796929" // TODO: remove traceId from props
```

Comment:
Can we delete commented out code thorough PR if they are unnecessary?

Reply:
It is props for demo page. so if you want to see capability of package, should use these props. I think we should keep them.